cmd/harmony/flags.go

```diff
@@ -397,6 +403,14 @@ func applyGeneralFlags(cmd *cobra.Command, config *harmonyconfig.HarmonyConfig)
 	if cli.IsFlagChanged(cmd, isBackupFlag) {
 		config.General.IsBackup = cli.GetBoolFlagValue(cmd, isBackupFlag)
 	}
+
+	if cli.IsFlagChanged(cmd, triesInMemoryFlag) {
+		value := cli.GetIntFlagValue(cmd, triesInMemoryFlag)
+		if value <= 1 {
+			panic("Must number greater than 1 for txpool.accountslots")
```

Comment:
Panic message refers to `txpool.accountslots`

Reply:
How did you decide that the minimum value should be 1 ?

Reply:
When triesInMemoryFlag is used by the `Stop` method, the following offsets are used to cache the blocks:
`for _, offset := range []uint64{0, 1, bc.cacheConfig.TriesInMemory - 1} {`
If `TriesInMemory` is set to `1`, the offset would become `0`. Wanted to prevent the repetition but I should be more clear on that. Let me also fix the panic message.